doc/index.rst

```diff
@@ -14,8 +14,6 @@
    :caption: Tutorials
 
    tutorials/01_getting_started/01_getting_started
```

Comment:
I made the same comment on @MattToast's recent PR: I think we should move away from numbered tutorials, just keep a clean list. Otherwise mixing numbered and unnumbered ones is a bit confusing.

Reply:
lets follow up on this one.